source/templates/workflows.slim

```diff
@@ -126,10 +126,12 @@
 												.parameter.description
 													p[ng-if="!output.title()"] == data[:strings][:workflows][:outputs][:no_description]
 													div[ng-if="output.title()"]
-														button[ng-click="workflowsCtrl.outputSelected(output)"]
+														button[ng-if="workflowsCtrl.hasOutputDescriptionOrSummary(output)" ng-click="workflowsCtrl.outputSelected(output)"]
 														.toggleabble-description
-															p[sanitized-markdown="output.title()" ng-class="{'oneliner': workflowsCtrl.openedOutputs.indexOf(output) == -1}"]
-															== svg("arrow-down")
+															p[sanitized-markdown="output.title()" ng-if="!workflowsCtrl.isOutputOpened(output)" ng-class="{'oneliner': workflowsCtrl.openedOutputs.indexOf(output) == -1}"]
+															p[sanitized-markdown="output.shortestDescription()" ng-if="workflowsCtrl.isOutputOpened(output)"]
```

Comment:
The two can be merged to one, with conditional content. Also, for the ng-class you can use the method you created earlier:
```suggestion
															p[sanitized-markdown="workflowsCtrl.isOutputOpened(output) ? output.shortestDescription() : output.title()" ng-class="{'oneliner': !workflowsCtrl.isOutputOpened(output)}"]
```